package.json

```diff
@@ -6,22 +6,24 @@
   "author": "ewolfe <e@ewolfe.me>",
   "license": "MIT",
   "scripts": {
-    "dev": "concurrently \"env-cmd .env micro-dev\" \"lt --subdomain prlint --port 3000\"",
-    "staging": "concurrently \"micro\" \"lt --subdomain prlint --port 3000\"",
-    "start": "micro",
+    "dev": "concurrently \"env-cmd .env micro-dev src/app.js\" \"lt --subdomain prlint --port 3000\"",
+    "staging": "concurrently \"micro src/app.js\" \"lt --subdomain prlint --port 3000\"",
+    "start": "micro src/app.js",
```

Comment:
Tweaks, since we're now exporting core functions alongside the app. If you think it's better to isolate core into it's own package, then we can revert these tweaks.

Reply:
Isolating core into it's own package sounds like a good idea, but this works for now 👍